packages/storybook/src/designsystem/Typography.stories.tsx

```diff
@@ -19,12 +17,21 @@ export function Intro() {
 }
 
 export function FontFamily() {
+  const context = useThemeUI()
+  const { theme } = useThemeUI()
+  console.log({ context })
+  console.log('test')
+  console.log({ theme })
```

Comment:
Do we need all this and the two calls to `useThemeUI`?

Reply:
my bad!  was just testing, thanks